examples/pcpChannelClient/README.md

```diff
@@ -1,30 +1,37 @@
+# PCP Example client
+
 This directory contains a tiny sample client for the pcpChannel WebSockets connection.
 
+## Starting up the GPII and trying out the PCP channel example client.
+
 It can be tested by firing up any configuration of the GPII which includes a local FlowManager. Particularly
 suitable are the mock configurations in %universal/gpii/configs/mocks - for example you can type
 
-    node gpii.js gpii/configs/mocks gpii.config.development.all.local.mock.windows
+    `node gpii.js gpii/configs/mocks gpii.config.development.all.local.mock.windows`
 
-or 
+or
 
-    node gpii.js gpii/configs/mocks gpii.config.untrusted.development.all.local.mock.windows
+    `node gpii.js gpii/configs/mocks gpii.config.untrusted.development.all.local.mock.windows`
 
-from the root of this repository.
+from the root of this repository. This will start up the servers in trusted or untrusted mode, respectively.
 
 After that, you can fire up the client at any time during the lifetime of the FlowManager by typing
 
-    node pcpChannelClient.js
+    `node pcpChannelClient.js`
 
 from this directory in another shell.
 
 Before or after that, you can experiment with logging in and out of the GPII using endpoints such as
 
-    http://localhost:8081/user/snapset_1a/login
+    `http://localhost:8081/user/snapset_1a/login`
 
 and
 
-    http://localhost:8081/user/snapset_1a/logout
+    `http://localhost:8081/user/snapset_1a/logout`
+
+Note that the NP sets used with the above configurations should be in the testData/preferences folder of this repository.
 
+## Example payloads received by the PCP
```

Comment:
It seems that the payloads that the PSP receives have been slightly changed:

- they now have a `type` and a `payload` property
- there is a new message which is sent when a setting has been applied

Can you please document these modifications?

Reply:
Ah - forgot to update that, cheers Georgi! This has been done in the below commit